tests/nixos/fetch-git/test-cases/fetchTree-shallow/default.nix

```diff
@@ -0,0 +1,45 @@
+{
+  description = "fetchTree fetches git repos shallowly by default";
```

Comment:
No need to make this a VM test. Just add it to `tests/functional`, similar to `tests/functional/fetchGit.sh`.

Reply:
please see comment below